zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/ElasticsearchEngineClient.java

```diff
@@ -155,7 +157,13 @@ private PutIndexTemplateRequest putIndexTemplateRequest(
                   t.aliases(indexTemplateDescriptor.getAlias(), Alias.of(a -> a))
                       .mappings(
                           deserializeJson(IndexTemplateSummary._DESERIALIZER, templateMappings)
-                              .mappings()))
+                              .mappings())
+                      .settings(
+                          s ->
+                              s.index(
+                                  i ->
+                                      i.numberOfShards(settings.numberOfShards)
+                                          .numberOfReplicas(settings.numberOfReplicas))))
```

Comment:
what operate currently does with these is it uses these settings to create a component template with a default name (in runtime) before creating index templates. All index templates will be `composedOf` this component template. then, we don't have to pass these configurations around

Reply:
I don't understand the need to have a component template, yes every index template composed_of the component template for settings (and the config doesn't have to be "passed around") 

but the same applies here, the config is not passed around. An index/templates replicas/shards are calculated once in the schema manager and then passed to the request.

Reply:
At some point we need to check if there is some per index replicas/shards settings, and if not we set it to some default I feel adding a component template is just redundant.

Reply:
we can leave it as is right now, and change later if needed.